backend/server/routers/movies.py

```diff
@@ -1,15 +1,22 @@
-from fastapi import APIRouter, Depends
-from pydantic import BaseModel
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel, Field
+from typing import Optional
 from ..util import DB, newId
 from ..dependencies import auth, admin
 
 router = APIRouter()
 
+# Temporary in-memory storage for reviews
```

Comment:
This won't work if the backend server restarts (which it does daily for performance reasons).